src/types/profile-derived.js

```diff
@@ -246,3 +246,11 @@ export type SelectedState =
  * to scroll the initially selected track into view once the page is loaded.
  */
 export type InitialSelectedTrackReference = HTMLElement;
+
+/**
+ * Page data for ProfileFilterNavigator component.
```

Comment:
No action needed: We should really create a `src/types/app.js` file for types like this. There are tons of component-specific types in `https://github.com/firefox-devtools/profiler/blob/a53a4247319b8685ca533fb4b19718887023bbe4/src/types/actions.js#L56` as well.